src/antsibull_changelog/update_galaxy.py

```diff
@@ -0,0 +1,26 @@
+# Author: Sagar Paul <paul.sagar@yahoo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023, Ansible Project
+
+"""
+Update the galaxy the yaml file.
+"""
+
+from __future__ import annotations
+
+from .errors import ChangelogError
+from .yaml import load_yaml, store_yaml
```

Comment:
(This might need to be reformatted with `nox -e formatters`)

```suggestion
from typing import TYPE_CHECKING

from .yaml import load_yaml, store_yaml

if TYPE_CHECKING:
   from _typeshed import StrOrBytesPath
```